### ISSUE TITLE
Labels in labels.json file

### ISSUE THREAD

**Issue by bhaswa** [SeanNaren/deepspeech.pytorch#513]
Hi,
I know that the first character in the json file "_" is considered as blank character. Is there any assumption on the second character " ' " also? When tried removing the second character and train the model, the training loss is not decreasing much. Also The WER is showing 100 after every epoch.

**Comment by raotnameh** [comment]
@bhaswa  did you also remove the " ' " character from the training transcripts?
